fre/analysis/freanalysis.py

```diff
@@ -0,0 +1,63 @@
+import click
+
+from .subtools import install_analysis_package, run_analysis, uninstall_analysis_package
```

Comment:
so these things will need separate, protected-entry points for `click` decorators in `fre/freanalysis/subtools.py` if module-driven usage is at all a design here (it absolutely should IMO).

In anycase, if you decorate the functions in `subtools` now as they are, `click` will block imports of those functions elsewhere.